README.md

```diff
@@ -810,6 +810,31 @@ oneview_user 'User1' do
 end
 ```
 
+### [oneview_id_pool](examples/id_pool.rb)
+
+```ruby
+oneview_id_pool 'IDPool1' do
+  client <my_client>
+  pool_type <pool_type> # String - The type of the pool. Values: (ipv4, vmac, vsn, vwwn)
+  enabled <enabled> # [TrueClass, FalseClass] - The status of the pool
+  action [:update]
+end
+
+oneview_id_pool 'IDPool1' do
+  client <my_client>
+  pool_type <pool_type> # String - The type of the pool. Values: (ipv4, vmac, vsn, vwwn)
+  count <count>         # Integer - The quantity of IDs to allocate
+  action :allocate_count
+end
+
+oneview_id_pool 'IDPool1' do
+  client <my_client>
+  pool_type <pool_type> # String - The type of the pool. Values: (ipv4, vmac, vsn, vwwn)
+  id_list <id_list>     # Array<String> - The IDs list (or IDs separeted by comma)
+  action [:allocate_list, :collect_ids]
+end
+```
+
```

Comment:
Please add also one block with the full description of the resource, i.e. with all the possible properties and actions listed